proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,113 @@
+
+``Contributor``
+~~~~~~~~~~~~~~~
+Any individual or organization ``user`` may become a ``contributor`` by submitting a pull request adding, correcting, or aliasing jurisdictional OCDIDs.
+First time ``contributors`` will be advised to consult with an existing ``committer`` before submitting their first PR.
+
+``Committer`` 
+~~~~~~~~~~~~~
+``Committers`` have the ability to approve and commit pull requests from ``contributors`` and other ``committers``. 
+An initial cohort of ``committers`` will be determined by current project contributors.
+Any individual or organization who is a ``contributor`` and who agrees to the repsonsibilities listed here may apply to become a ``committer``. 
+
+* There will be a two month period where existing ``committers`` may raise objections to an applicant, otherwise the applying organization/individual will be considered approved.
+* In the case that an objection is raised, a discussion and vote will be had amongst existing ``committers``, with a simple majority necessary for approval.
+
+``Committers`` will be approved on an organizational basis, unless explicitly noted for an individual.
+
+* Multiple members of an organization may have committing ability, but the organization itself will be counted for all voting and approval processes.
+* Organizations will be asked to designate a primary point of contact who will act on behalf of the organization in procedural matters.
+
+``Committers`` are expected to participate in approval, support, maintenance, and other community related activities or may have their ``committer`` status revoked. 
+
+* ``Committers`` will have their status reviewed on a calendar year basis. 
+
+
+Contribution Process
+--------------------
+General contributions
+~~~~~~~~~~~~~~~~~~~~~
+Any ``contributor`` may create a pull request for generative IDs, corrections, aliases, etc. 
+
+* A pull request from a ``contributor`` will be considered accepted when reviewed by and approved by two ``committers``.
+* If the contribution is from a current ``committer`` in good standing, only one additional ``committer`` review is necessary.
+* No two members of the same organization may be involved in the acceptance of a pull request.
+
+Commits should be reviewed within 2 weeks of a pull request. Accellerated timeline needs should be communicated via Slack/email.
+
+* If, after two months there is neither a formal approval or an ongoing conversation around a request, that request will be considered accepted.
+* If a conversation around a request cannot reach consensus, after two months the ``contributor`` may request a final vote from the cohort of existing ``committers``.
+
+Approval will be handled on a per-file, rather than a per-commit, basis.
+
+New commits should include have a well-formed explanations, especially if generating new IDs or types.
+
+Formalized guidelines for approval will be created for use by ``committers``. These guidelines will include:
```

Comment:
We'll these be automated checks.

Reply:
I would love for as many as possible to be, and think that there is a lot of opportunity for automated validation. I think there's always going to be a little bit of need for a human touch though, especially for things like determining whether a new flavor of special district is distinct enough to deserve its own type and the like. I also think there should still be more rigorous guidelines written before these automated checks get implemented.